Déplacement du tuto "Duniter sur RPi 3 avec Yunohost derrière Freebox v5" vers le wiki

blaurent49 opened the pull request:
Comme suggéré après l'autre pull request, je propose le transfert du tuto vers le wiki.

blaurent49 committed:
Add files via upload

Tuto d'installation de Duniter avec Yunohost derrière Freebox v5

blaurent49 committed:
Create temp.txt

blaurent49 committed:
Add files via upload

blaurent49 committed:
Delete temp.txt

blaurent49 committed:
Update wiki.md

blaurent49 committed:
Update wiki.md

c-geek commented:
Bien, par contre 2 choses à modifier pour que ça passe correctement : 

* les images devraient être stockées non pas dans `content/...` mais dans  `images/wiki/nom_article`
* les URL d'images être de la forme `../../images/nom_article/nom_image.png` étant donné la position de l'article dans l'arborscence

Tu peux vérifier que ça fonctionne bien en buildant le site comme indiqué dans le README, ou tenter à l'aveugle mais alors on devra peut-être corriger après coup.

blaurent49 committed:
Create temp.txt

blaurent49 committed:
Add files via upload

Upload des captures d'écran relatives au tuto d'installation de Duniter sur RPi derrière Freebox v5

blaurent49 committed:
Delete temp.txt

blaurent49 committed:
Update install-duniter-freebox_v5-yunohost.md

Modification des chemins vers les images.

blaurent49 committed:
Delete freebox_v5_baux_dhcp_perm.PNG

blaurent49 committed:
Delete freebox_v5_redirections_ports.PNG

blaurent49 committed:
Delete freebox_v5_routeur.png

blaurent49 committed:
Delete gestion_domaine.PNG

blaurent49 committed:
Delete gestion_sous_domaine.PNG

blaurent49 committed:
Delete gestion_sous_domaine_2.PNG

blaurent49 committed:
Delete yunohost_certificat_ssl.PNG

blaurent49 commented:
> Tu peux vérifier que ça fonctionne bien en buildant le site comme indiqué dans le README, ou tenter à l'aveugle mais alors on devra peut-être corriger après coup.

J'ai effectué les modifications. Par contre, je n'ai pas réussi en buildant le site localement. Les commandes `virtualenv ` et suivantes sont inconnues. Pourtant Python 2.7.3 est installé sur ma machine. Qu'ai-je raté ?

Je ne suis vraiment pas sûr de pouvoir m'en occuper d'ici 10 jours désormais (peut-être demain mais vraiment pas sûr), donc on y va à l'aveugle ? J'ai essayé d'être vigilant sur mes chemins relatifs mais bon...

Insoleet commented:
Je merge et corrige ce qui ne va pas (problème de Titre sur l'article notamment).

Insoleet commented:
Et voilà, ton article est intégré ! Merci beaucoup ! https://duniter.org/fr/wiki/duniter/install-duniter-freebox_v5-yunohost

blaurent49 commented:
> Et voilà, ton article est intégré ! 
Cool ! 

> Merci beaucoup ! 
Avec plaisir. Vous avez déjà fait un super travail. J'essaie de contribuer modestement.
